CHANGELOG.md

```diff
@@ -14,7 +14,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Security
   -->
 
-## [Unreleased]
+## [2.0.0] unreleased
+
+### Changed
+* #1093 Changed to implement [hashed](https://docs.djangoproject.com/en/stable/topics/auth/passwords/)
```

Comment:
```suggestion
* #1093 (**Breaking**) Changed to implement [hashed](https://docs.djangoproject.com/en/stable/topics/auth/passwords/)
```

I understand it already said "Breaking change" but I typically like it when I can see it immediately. Just personal opinion and no need to merge.